src/main/java/com/alkemy/ong/mapper/CategoryMapper.java

```diff
@@ -0,0 +1,28 @@
+package com.alkemy.ong.mapper;
+
+import com.alkemy.ong.domain.Category;
+import com.alkemy.ong.repository.model.CategoryModel;
+import org.springframework.stereotype.Component;
+
+@Component
+public class CategoryMapper {
+
+    public Category categoryModel2Category (CategoryModel categoryModel){
+        Category category= Category.builder().id(categoryModel.getId()).
```

Comment:
Dejar un espacio antes y después del `=`. En cada línea pongamos un solo atributo y que empiece con punto, es decir:
```suggestion
        Category category= Category.builder().id(categoryModel.getId())
        .name(categoryModel.getName())
```
Fijate que de esta forma, el punto está al principio del renglón y no al final 👌🏼